recipe/meta.yaml

```diff
@@ -105,48 +107,47 @@ requirements:
   # things do not work. To get all the standard dependencies of Sage, install
   # the "sage" package (which requires this "sagelib" package.)
   run:
-    - arb 2.11.*
+    - arb 2.13.*
     - blas 1.1 {{ variant }}
     - openblas 0.2.20|0.2.20.*
     - brial 1.2.*
     - cliquer 1.21
-    - cypari2 1.1.*
-    - cysignals 1.6.*
-    - cython >=0.27
+    - cypari2 1.1.4
+    - cysignals 1.7.*
+    - cython >=0.28
     - ecl 16.1.*
     - eclib >=20171002
     - ecm 7.*
-    - fflas-ffpack >=2.2.2,<2.3.0
     - future
-    - givaro >=4.0.0,<4.0.3
```

Comment:
Why did you remove givaro and fflas-ffpack?

Reply:
Because they are pulled in by linbox in the correct versions.

Reply:
Yes, but they need to be pinned here (at least have them in host for run_exports to work) since sagelib links directly to givaro. Otherwise if we upgrade givaro version in linbox, sage will use that version of linbox and the newer givaro, but that'll not work as the already built sage expects the older version of givaro

Reply:
Right. Sorry for that.